### PR TITLE
Assignment 1a

### PR DESCRIPTION
**Pull request by TechGlue**
Added the changes to SecretSantaApi and screenshot showcasing changes!  
image

### CODE REVIEW

**Comment by Ashrom0** [comment]
I will review this

**Review by Ashrom0 — APPROVED** [review]
API project updated to output new welcome message in second pull request: ✔
Screenshot of web browser running the API project included in second pull request: ✔
Both new output and screenshot is seen. Good work.

**Review by Ashrom0 — COMMENTED** [review]
API project updated to output new welcome message in second pull request: ✔
Screenshot of web browser running the API project included in second pull request: ✔
Both new output and screenshot is seen. Good work.

File: SecretSanta/src/SecretSanta.Api/Startup.cs [hunk 1]
```diff
@@ -32,7 +32,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 endpoints.MapGet("/", async context =>
                 {
-                    await context.Response.WriteAsync("Hello from API!");
+                    await context.Response.WriteAsync("This is going to be a great quarter!!!");
```
Comment (mmwoodfo) [inline SecretSanta/src/SecretSanta.Api/Startup.cs:35]: :smiley:

**Pull request closed without merging**